outbound.js

```diff
@@ -1,13 +1,14 @@
+    var _ = require('underscore');
 
     var BASE_URL = "https://api.outbound.io/v2";
     var VERSION = '0.3.0';
```

Comment:
Change version here too. Would be even better if you could find a way to parse the package.json file and get the version number there. That way we only have to have it in a single place.